wire-ios-request-strategy/Sources/E2EIdentity/ACME/AcmeAPITests.swift

```diff
@@ -224,6 +222,38 @@ class AcmeClientTests: ZMTBaseTest {
         }
     }
 
+    func testThatItSendsChallengeRequest() async throws {
+        // expectation
+        let headerNonce = "ACMENonce"
+        let expectation = ChallengeResponse(type: "JWT",
+                                            url: "https://acme.example.com/acme/provisioner1/challenge/foVMOvMcap/1pceubr",
+                                            status: "pending",
+                                            token: "NEi1HaRRYqM0R9cGZaHdv0dBWIkRbyCY",
+                                            nonce: headerNonce)
+
+        // given
+        let path = "https://acme.example.com/acme/provisioner1/challenge/foVMOvMcapXlWSrHqu4BrD1RFORZOGrC/1pceubrFUZAvVQI5XgtLDMfLefhOt4YI"
+
+        // mock
+        let mockResponse = HTTPURLResponse(
+            url: URL(string: path)!,
+            statusCode: 200,
+            httpVersion: "",
+            headerFields: ["Replay-Nonce": headerNonce]
+        )!
+        let challengeResponseData = try! JSONEncoder.defaultEncoder.encode(expectation)
```

Comment:
remove force unwrapping to ensure tests don't crash
```suggestion
        let mockResponse = try XCTUnwrap(HTTPURLResponse(
            url: URL(string: path)!,
            statusCode: 200,
            httpVersion: "",
            headerFields: ["Replay-Nonce": headerNonce]
        ))
        let challengeResponseData = try JSONEncoder.defaultEncoder.encode(expectation)
```